ghmirror/core/mirror_requests.py

```diff
@@ -98,6 +98,15 @@ def _online_request(method, url, cached_response,
         cached_response.headers['X-Cache'] = 'API_TIMEOUT_HIT'
         return cached_response
 
+    except requests.exceptions.ConnectionError:
```

Comment:
Why not catch `requests.exceptions.RequestException`? Guess there are plenty of other error cases.

Reply:
Because you don't want to serve from cache in every case, e.g. `HTTPError`

https://docs.python-requests.org/en/latest/_modules/requests/exceptions/